modules/ossm-cr-status.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-status_{context}"]
+= Service Mesh status
+
+The information in your `ServiceMeshControlPlane` file under `status` describes the current state of your service mesh.  This information is generated by the Operator and is read-only.
+
+.`ServiceMeshControlPlane` resource `ControlPlaneStatus`
+|===
+|Name |Description |Type
+
+|`observedGeneration`
+|The generation observed by the controller during the most recent reconciliation. The information in the status pertains to this particular generation of the object. The `status.conditions` are not up-to-date if the `status.observedGeneration` field doesn't match `metadata.generation`.
+|integer
+
+|`annotations`
+|The `annotations` parameter stores additional, usually redundant status information, such as the number of components deployed by the `ServiceMeshControlPlane`. These statuses are used by the command line tool, `oc`, which does not yet allow counting objects in JSONPath expressions.
+|Not configurable
+
+|`readiness`
+|The readiness status of components & owned resources.
+|string
+
+|`operatorVersion`
+|The version of the operator that last processed this resource.
+|string
+
+|`components`
+|Shows the status of each deployed control plane component.
+|string
+
+|`appliedSpec`
+|The resulting specification of the configuration options after all profiles have been applied.
+|`ControlPlaneSpec`
+
+|`conditions`
+|Represents the latest available observations of the object's current state. `Reconciled` indicates whether the operator has finished reconciling the actual state of deployed components with the configuration in the `ServiceMeshControlPlane` resource. `Installed` indicates whether the control plane has been installed. `Ready` indicates whether all control plane components are ready.
```

Comment:
```suggestion
|Represents the latest available observations of the object's current state. `Reconciled` indicates that the operator has finished reconciling the actual state of deployed components with the configuration in the `ServiceMeshControlPlane` resource. `Installed` indicates that the control plane has been installed. `Ready` indicates that all control plane components are ready.
```

Reply:
I think "that" maybe makes more sense here, since "whether" implies that the status will also tell you if the opposite is true, e.g. the `Ready` status can also indicate if something isn't ready. Presumably that's not the case though and there would be a `NotReady` status, for example.